### DISCUSSION TITLE
各位开发者注意 ⚠️

### DISCUSSION THREAD

**Discussion by wyc-26** [share121/inter-knot#4792] [科技]
![IMG_6426](https://github.com/user-attachments/assets/3b831531-1616-48b6-8172-0374e4b30918)
我是当事人小号，现来绳网提醒一下大家
虽然这件事差不多过去了，该封的也都封了
但还是请大家提高警惕，不要下载未知链接里的内容
（本人没有感染病毒，别误会了）

**Comment by xiaqizhengzhao** [comment]
惨
